lab_1_classify_by_unigrams/main.py

```diff
@@ -5,9 +5,22 @@
 """
 # pylint:disable=too-many-locals, unused-argument, unused-variable
 
+# откуда берется language(3) + округлить мсе + closest profile:/
+
 
 def tokenize(text: str) -> list[str] | None:
-    """
+    if isinstance(text, str):
```

Comment:
revert the docstring

Reply:
use early return - https://medium.com/swlh/return-early-pattern-3d18a41bba8

Reply:
done